src/options/options.routes.ts

```diff
@@ -117,10 +117,6 @@ import {
   GetAccountPremiumResponse,
   GetAccountFeesBaseResponse
 } from './options.requests';
-// import {
-//   validateMintRequest,
```

Comment:
Have we decided against eventually trying to make these work and bring them back? Or should we log a card to do that? Either way, agree with the choice to delete this comment

Reply:
Added an issue card for it.